common/lib/protocol-definitions/src/clients.ts

```diff
@@ -5,53 +5,137 @@
 
 import { IUser } from "./users";
 
+/**
+ * A client's connection mode - either view-only ("read") or allowing edits ("write").
+ *
+ * @remarks Note: this should not be confused with user permissions.
+ * If a user does not have write privileges, they will still not be permitted to make edits, even if the
+ * connection mode is set to "write".
```

Comment:
that's incorrect. If user has no write permissions, user can't get "write" connection.
It's the opposite that confuses people ("read" connection does not mean much)

Reply:
I see. I'll update accordingly. Thanks!